### ISSUE TITLE
fetch() doesn't work with nativescript-plugin-firebase

### ISSUE THREAD

**Issue by zhengiko** [EddyVerbruggen/nativescript-plugin-firebase#42]
Hi Guys! Thank you for excellent plugin - it's really useful. But I've got a problem:

I'm using **fetch()** in my project for make requests to some geo services. (google places, google geocoder, nominatim). And this method doesn't work with firebase plugin. For example, this code (url from nominatim wiki) must show me in console json response:
```
fetch("http://nominatim.openstreetmap.org/search/Unter%20den%20Linden%201%20Berlin?format=json&addressdetails=1&limit=1&polygon_svg=1")
    .then(function(response) {
      return response.json();
    }).then(function(data) {
      console.log("##############################")
      console.log("###=" + JSON.stringify(data))
      console.log("##############################")
    });
```
and I've got nothing if I add firebase in my project or if I add this code in firebase demo project. Need help! Thanks.

**P.S.**
With **http.getJSON()** the same situation. 
```
var http = require("http");
http.getJSON("http://nominatim.openstreetmap.org/search/Unter%20den%20Linden%201%20Berlin?format=json&addressdetails=1&limit=1&polygon_svg=1").then(function (data) {
	console.log("##############################")
	console.log("###=" + JSON.stringify(data))
	console.log("##############################")
}, function (e) {
	console.log(e);
});
```

**Comment by zhengiko** [comment]
It works fine if I use old version (2.1.8)
But with google-services I promblems

**Comment by EddyVerbruggen** [comment]
Do you see anything in the log?

**Comment by zhengiko** [comment]
When I use fetch() I see nothing in console.

**Comment by EddyVerbruggen** [comment]
And this is on iOS, Android, or both?

**Comment by zhengiko** [comment]
didn't test iOS. I work at the moment only with Android.

**Comment by EddyVerbruggen** [comment]
I just tested this again on Android without problems. I'm running `init` after the app started and added your fetch code in its success callback function. Then I logged in via Firebase Auth and queried the database. No problems.

Perhaps something was fixed in 3.5.0.

I'm closing this, but feel free to send me your project if you want me to take a deeper look.
